<ui version="4.0" >
 <class>Value</class>
 <widget class="QDialog" name="Value" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Asignar valor a registro</string>
  </property>
  <widget class="QLabel" name="direcLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Direcci√≥n o nombre de registro:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="direcLineEdit" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Valor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLineEdit" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="aceptarButton" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarButton" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
